tensorrec/representation_graphs.py

```diff
@@ -58,9 +58,38 @@ def connect_representation_graph(self, tf_features, n_components, n_features, no
         return normalized_repr, weights_list
 
 
+class IdentityRepresentationGraph(AbstractRepresentationGraph):
+    """
+    Uses the features as the representation. This representation graph does no transformation to the features.
+    """
+
+    def connect_representation_graph(self, tf_features, n_components, n_features, node_name_ending):
+
+        if n_components != n_features:
+            raise ValueError('{} requires n_features and n_components to be equal. Either adjust n_components or use a '
+                             'different representation graph.'.format(self.__class__.__name__))
+
+        return tf.sparse_tensor_to_dense(tf_features, validate_indices=False), []
+
+
+class WeightedIdentityRepresentationGraph(IdentityRepresentationGraph):
```

Comment:
@jcauteru Given that this repr is weighting the features, should I remove `Identity` from the name? Inherently, it isn't really an identity.

Reply:
The name to me implies that the identity _is_ your representation, not that you're applying the identity to the input. Also, you're not even actually matmul by I anymore lol so even less true!

:+1: on name change

Maybe `FeaturePassThroughRepresentationGraph` or something?